NGTrackForce/src/app/components/associate-list-page/associate-list-page.component.ts

```diff
@@ -123,8 +142,13 @@ export class AssociateListPageComponent implements OnInit, OnDestroy, AfterViewI
 
   ngOnDestroy(): void {
     //Called once, before the instance is destroyed.
+   this.lsHelp.removeStorageItem("checked");
```

Comment:
why do this?

Reply:
local window storage needs global helper